src/Raven.Server/Documents/Handlers/ReplicationHandler.cs

```diff
@@ -305,6 +305,7 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
                     {
                         ["Key"] = new DynamicJsonValue
                         {
+                            [nameof(item.Key)] = item.Key.GetType(),
```

Comment:
`GetType` ?

Reply:
yeah, it can be any type that inherit from the `ReplicationNode` class

Reply:
added `ToString()`